docs/json-schema/comment.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$id": "https://sublinks.org/comment.schema.json",
+  "title": "Comment",
+  "description": "A comment in a Sublinks Post attributable to an author.",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "The unique identifier for the comment.",
+      "type": "string"
+    },
+    "post": {
```

Comment:
Wdyt about post_id instead? Or should it be post_url?

Reply:
I'm going to rename this to `post_id` I think that I forgot to update it here 😆